PROTO_tests/tests/operator_test.py

```diff
@@ -0,0 +1,774 @@
+        f_arr = ak.linspace(1, 5, 10)
+        b_arr = i_arr % 2 == 0
+        u = np.uint(7)
+        i = 7
+        f = 3.14
+        b = True
+
+        # test uint opequals uint functionality against numpy
+        for arr in u_arr, bi_arr, np_arr:
+            arr += u
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr += arr
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr -= u
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr -= arr
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        # redeclare minus self zeroed out
+        u_arr = ak.array(
+            [0, 1, 2, 3, 4, 2**64 - 5, 2**64 - 4, 2**64 - 3, 2**64 - 2, 2**64 - 1],
+            dtype=ak.uint64,
+        )
+        bi_arr = ak.array(
+            [0, 1, 2, 3, 4, 2**64 - 5, 2**64 - 4, 2**64 - 3, 2**64 - 2, 2**64 - 1],
+            dtype=ak.bigint,
+            max_bits=64,
+        )
+        np_arr = np.array(
+            [0, 1, 2, 3, 4, 2**64 - 5, 2**64 - 4, 2**64 - 3, 2**64 - 2, 2**64 - 1],
+            dtype=np.uint,
+        )
+        for arr in u_arr, bi_arr, np_arr:
+            arr *= u
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr *= arr
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr **= u
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr **= arr
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr %= u
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        for arr in u_arr, bi_arr, np_arr:
+            arr //= u
+        assert u_arr.to_list() == np_arr.tolist()
+        assert bi_arr.to_list() == np_arr.tolist()
+
+        # redeclare divide zeroed out
+        u_arr = ak.array(
+            [0, 1, 2, 3, 4, 2**64 - 5, 2**64 - 4, 2**64 - 3, 2**64 - 2, 2**64 - 1],
+            dtype=ak.uint64,
+        )
+        bi_arr = ak.array(
+            [0, 1, 2, 3, 4, 2**64 - 5, 2**64 - 4, 2**64 - 3, 2**64 - 2, 2**64 - 1],
+            dtype=ak.bigint,
+            max_bits=64,
+        )
+        np_arr = np.array(
+            [0, 1, 2, 3, 4, 2**64 - 5, 2**64 - 4, 2**64 - 3, 2**64 - 2, 2**64 - 1],
+            dtype=np.uint,
+        )
```

Comment:
Again, maybe a helper function just to reduce the number of places required for potential updates in the future?